reactDashboard/src/pages/Accessibility.jsx

```diff
@@ -24,6 +25,7 @@ const Accessibility = () => {
         </ul>
       </div>
       <div className="accessibility-details">
+        <hr className="full-width-hr" />
```

Comment:
For the whole page: having a "full-width-hr" before and after each heading is really jarring to me on dark mode. I find it very distracting and my eyes don't flow to the text beneath the heading. It's not as jarring on light mode but possibly still unnecessary. My suggestion is to remove the one after each of the three headings so the sections are clearly separated.

Reply:
✅